torch/nn/_intrinsic/qat/modules/conv_relu.py

```diff
@@ -1,7 +1,6 @@
 from __future__ import absolute_import, division, print_function, unicode_literals
-from torch.nn.qat import Conv2d as QATConv2d
-from torch.nn._intrinsic import ConvReLU2d as NNConvReLU2d
-from torch.quantization.QConfig import default_qat_qconfig
+from ....qat.modules.conv import Conv2d as QATConv2d
+from ...modules.fused import ConvReLU2d as NNConvReLU2d
```

Comment:
NIT: Why have conv_bn and conv_bn_relu in one file and conv_relu in another. Will it be better to have them in one file?

Reply:
sure